load.js

```diff
@@ -14,23 +14,36 @@ const ABI = process.versions.modules
 const inWebpack = typeof __webpack_require__ === 'function'
 const runtimeRequire = inWebpack ? __non_webpack_require__ : require
 
-function load (name) {
-  const root = __dirname
-  const build = `${root}/build/Release/${name}.node`
-
+function maybeLoad (name) {
   try {
-    return runtimeRequire(build) || runtimeRequire(find(root, name))
+    return load(name)
   } catch (e) {
     // Not found, skip.
   }
 }
 
-function find (root, name) {
+function load (name) {
+  const filename = find(name)
+
+  if (!filename) {
+    throw new Error(`Could not find a ${name} binary for ${PLATFORM}${LIBC}-${ARCH}.`)
+  }
+
+  return runtimeRequire(filename)
+}
+
+function find (name, binary = false) {
```

Comment:
LGTM. In which situations `binary` is used?

Reply:
It will be used by the crashtracker since it's split between a library and a sidecar binary. I'll add support for WASM eventually as well, and extract this code to a separate repo.